dali/operators/reader/loader/video/frames_decoder.cc

```diff
@@ -106,6 +148,46 @@ FramesDecoder::FramesDecoder(const std::string &filename)
   DetectVfr();
 }
 
+
+
+FramesDecoder::FramesDecoder(char *memory_file, int memory_file_size)
+  : av_state_(std::make_unique<AvState>()),
+    memory_video_file_(MemoryVideoFile(memory_file, memory_file_size)) {
+  av_log_set_level(AV_LOG_ERROR);
+
+  av_state_->ctx_ = avformat_alloc_context();
+  DALI_ENFORCE(av_state_->ctx_, "Could not alloc avformat context");
+
+  const int av_io_buffer_size = 32768;
```

Comment:
Where does this number come from? 
And maybe write it as (1 << 15) so it's less magical?

Reply:
This was the value that FFMPEG sets as a default when you do not try to change that. I extracted it as a const field in the class, changed the value and added some comment.